cloud/installation/poller-troubleshoot.md

```diff
@@ -0,0 +1,264 @@
+
+### Step 1: Test Gorgone's Connection to the platform
+
+Check that Gorgone can connect to your Centreon Cloud platform.
+
+```shell
+nc -zv -w 5 gorgone-<organization_code>.euwest1.centreon.cloud 443
+```
+
+A message like this one should be printed:
+
+```shell
+Ncat: Connected to n.n.n.n:443.
+```
+
+If you get a timeout instead, this means that the network configuration doesn't allow the poller to talk to the platform. Check whether you need a proxy, or whether you need to [open some flows](prerequisites.md#network) or [allow some IP ranges](prerequisites.md#allowing-traffic-tofrom-aws-ip-ranges).
+
+### Step 2: Check that Gorgone is running
+
+Run the following command:
+
+```shell
+systemctl status gorgoned
+```
+
+If the status is not **active (running)**, restart Gorgone:
+
+```shell
+systemctl restart gorgoned
+```
+
+### Step 3: Check Gorgone logs
+
+Gorgone logs are written in the following file: check them for any errors.
+
+```shell
+/var/log/centreon-gorgone/gorgoned.log
+```
+
+### Step 4: Set Gorgone log level to info or debug
+
+The default log level for Gorgone is **error**. If you need more details about what Gorgone is doing:
+
+1. Edit the following file:
+
+   ```shell
+   /etc/sysconfig/gorgoned
+   ```
+
+2. Change the **--severity** parameter value to **info** or **debug**.
+
+3. Restart the **gorgoned** process:
+
+   ```shell
+   systemctl restart gorgoned
+   ```
+
+### Step 5: Check Gorgone's connection
+
+Once you have set the log level to info or debug, this INFO message should appear in Gorgone logs:
+
+```shell
+INFO - [pullwss] websocket connected
+```
+
+If you get another message, this means that there is a problem with the network configuration or an authentication problem.
+
+### Step 6: Check that the deploy command works
+
+Check the log messages generated by Gorgone while the configuration is being deployed. The log should contain the following lines, which mean that the poller has received the configuration.
+
+```shell
+2024-09-13 14:28:50 - INFO - [action] Copy processing - Received chunk for '/etc/centreon-engine//'
+2024-09-13 14:28:51 - INFO - [action] Copy processing - Copy to '/etc/centreon-engine//' finished successfully
+2024-09-13 14:28:51 - INFO - [action] Copy processing - Received chunk for '/etc/centreon-broker/'
+2024-09-13 14:28:51 - INFO - [action] Copy processing - Copy to '/etc/centreon-broker/' finished successfully
+```
+
+If all previous steps were OK but the log does not show that the poller has received the configuration, contact [our support team](https://support.centreon.com/hc/en-us).
+
+### Step 7: Check that configuration files are correctly updated
+
+Check the modification dates of configuration files: if you see that the files were not updated when you deployed the configuration, check that the **gorgone** user has rights on this folder.
+
+```shell
+ls -l /etc/centreon-engine
```

Comment:
@Nohzoh: shouldn't this be `ls -ld`?

Reply:
Also, we need to document the fix: explain the owner/permissions for that directory

Reply:
The `ls -l` will display all the configuration files written in the folder, it can be interesting to view them all.
However, we could use `ls -al` to display also the folder itself.
```suggestion
ls -al /etc/centreon-engine
```